info.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "dnstwist",
+  "version": "1.0.0",
+  "label": "dnstwist",
+  "description": "Domain name permutation engine for detecting homograph phishing attacks,typosquatting, fraud and brand impersonation.",
+  "publisher": "Fortinet",
```

Comment:
let's publish first version as community till the qualification is waiting in QA pipeline